ContextDesktop/src/main/java/com/github/migi_1/Context/utility/ProjectAssetManager.java

```diff
@@ -0,0 +1,58 @@
+package com.github.migi_1.Context.utility;
+
+import com.jme3.asset.AssetManager;
+
+/**
+ * This class makes the asset manager accessible from any class, after it has been initialized.
+ *
+ * SINGLETON class
+ * @author Marcel
+ *
+ */
+public final class ProjectAssetManager {
+
+    /**
+     * Object instance.
+     */
+    private  static ProjectAssetManager instance;
+
+    /**
+     * Instance of the Asset Manager.
+     */
+    private AssetManager assetManager;
+
+    /**
+     * Private singleton constructor.
+     */
+    private ProjectAssetManager() { }
+
+    /**
+     * Singleton initialisation method.
+     * @return newly created instance
+     */
+    public static ProjectAssetManager getInstance() {
+        if (instance == null) {
+            instance = new ProjectAssetManager();
+        }
```

Comment:
Instead of initialising the instance here, you can do it on the same line as the attribute declaration:

private static final ProjectAssetManager intance = new ProjectAssetManager()

Doing this gets rid of the if statement.